examples/rock_paper_scissors.py

```diff
@@ -1,18 +1,21 @@
-from argparse import ArgumentParser
+#!/usr/bin/env python
```

Comment:
do we really need this line?

Reply:
This allows you to run `./examples/rock_paper_scissors.py`.

I think that when you have a line `if __name__ == '__main__': ...` you should make it a proper executable.

Reply:
fair enough.